helm/scylla-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+
+The Scylla Operator has been installed. Check its status by running:
+
+  kubectl -n {{ .Values.namespace }} get pods -l "app.kubernetes.io/name={{ template "scylla-operator.name" . }}"
+
+Visit https://github.com/scylladb/scylla-operator for tutorials on how to
+create and configure Scylla clusters using the Scylla Operator and set up monitoring.
```

Comment:
Would be nice here to have the notes also include a port forward command for hooking up locally

Reply:
Currenly we support only cases where application using Scylla is deployed on k8s next to Scylla. This doesn't mean it's not possible, although we don't have enough generic solution to include it here. So I thought adding a link to repo itself where we also have docs, would be the best here.

Reply:
I was thinking more for debugging and development. Doing a simple port forward should work for this

```
kubectl port-forward --namespace <the-namespace> svc/scylla 9042:9042
```

Putting this in the notes would be helpful as a copy/paste